init/init-iptables.sh

```diff
@@ -65,6 +65,7 @@ iptables -t nat -A PROXY_REDIRECT -p tcp --dport "9200" -j ACCEPT # elastic
 iptables -t nat -A PROXY_REDIRECT -p tcp --dport "9300" -j ACCEPT # elastic
 iptables -t nat -A PROXY_REDIRECT -p tcp --dport "10000" -j ACCEPT # radiusconfd
 iptables -t nat -A PROXY_REDIRECT -p tcp --dport "10080" -j ACCEPT # byod
+iptables -t nat -A PROXY_REDIRECT -p tcp --dport "10500" -j ACCEPT # deviced/proxylistener
```

Comment:
Is this a REST port ?

Reply:
this is a coral port listen on a vrf interface for access from proxy tunnel